data.sql

```diff
@@ -70,3 +70,9 @@ INSERT INTO visits (animals_id, vets_id, date_of_visit)
 	(9, 2, '2020-08-03'),
 	(10, 3, '2020-05-24'),
 	(10, 1, '2021-01-11');
+
+-- This will add 3.594.280 visits considering you have 10 animals, 4 vets, and it will use around ~87.000 timestamps (~4min approx.)
```

Comment:
**[Optional]**

- [ ] This is a pair programming project but i cannot see any commit from your partner side. Kindly follow the requirements :blush:

Reply:
Thank you, we shall fix that in the next request.